systemtest/build.xml

```diff
@@ -40,6 +40,24 @@
 			<arg value="https://github.com/AdoptOpenJDK/openjdk-systemtest.git" />
 			<arg value="${SYSTEMTEST_ROOT}/openjdk-systemtest" />
 		</exec>
+		<if>
+			<equals arg1="${TARGET}" arg2="openj9" />
+			<then>
+				<antcall target="clone_openj9-systemtest" inheritall="true" />
+			</then>
+		</if>
+	</target>
+	
+	<target name="clone_openj9-systemtest">
+		<exec executable="git" failonerror="false">
+			<arg value="clone" />
+			<arg value="--depth" />
+			<arg value="1" />
+			<arg value="-b" />
+			<arg value="exclude_openj9.test.sharedCLasses.jvmti" />
+			<arg value="https://github.com/Mesbah-Alam/openj9-systemtest.git" />

```

Comment:
Is this a temporarily solution using personal repo?

Reply:
Yes. Please ignore it for now. I am only using it for testing using my personal builds. I will remove it before delivering.